pkgs/development/tools/jnv/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, autoconf
+, automake
+, clang
+, libclang
+, libtool
+}:
+rustPlatform.buildRustPackage rec
+{
+  pname = "jnv";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "ynqa";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-LN42UyXOStwgj/bbHl2MK1jTsDACTjDo+QaoZ9FzurA=";
+  };
+
+  cargoHash = "sha256-ZCYDlvKSuEONPSz/TS00mYolPnMYxsOwE9a0vK36UQQ=";
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    clang
+    libclang
+    libtool
+  ];
+
+  cargoPatches = [
```

Comment:
Maybe it's better to wait for https://github.com/ynqa/jnv/commit/57a8e9d01d3019aeb5b86fe761f6e7014939667a#r140025348?